Sources/WebAuthn/Helpers/Duration+Milliseconds.swift

```diff
@@ -0,0 +1,24 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the WebAuthn Swift open source project
+//
+// Copyright (c) 2022 the WebAuthn Swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of WebAuthn Swift project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+extension Duration {
+    /// The value of a positive duration in milliseconds, suitable to be encoded in WebAuthn types.
+    ///
+    /// - Note: If duration is less than zero, milliseconds will be capped to 0.
+    var milliseconds: UInt32 {
```

Comment:
I figured since it was internal, there was little harm to it being highly specific. If not, what about `clampedMilliseconds` since its clamped to 0 or larger?

Reply:
Yeah my concern is future devs assuming this works without reading it properly (and when I say future devs I of course mean me 🤣)

Clamped sounds good

Reply:
Turns out that negative number support was straightforward (and simpler), and added tests to verify. Do you think we need to make sure it's positive before encoding? Since it's just a hint, I don't think the browser will care.

Reply:
Nah I think if a dev starts trying to use a negative timeout that's on them